src/snaskii.c

```diff
@@ -215,19 +215,45 @@ void showscene(char scene[][NROWS][NCOLS], int number, int menu) {
 
 /* Instantiate the snake and a set of energy blocks. */
 
+
+struct _energy_block spawn_energy_block() {
+	struct _energy_block block;
+	block.x = rand() % NCOLS;
+	block.y = rand() % NROWS;
+	return block;
+}
+
 #define BLOCK_INACTIVE -1;
 
+/* Read sizeof(unsigned int) bytes from /dev/urandom */
+unsigned int generate_seed() {
```

Comment:
I'm not sure if this function belongs to this file.

Reply:
I thought about making a "utils" file, but didn't think it would be great to make a object file just for this function.